CONTRIBUTING.md

```diff
@@ -42,9 +43,54 @@ When creating a new issue, please add specific steps to reproduce the problem, u
 
 - WordPress - 5.0
 
+  You don't need to explicitly install WordPress if you use the provided, Docker-based wp-env.
+
+- Docker 
+
+  Docker installation depends on your OS. [Please follow their official instructions](https://docs.docker.com/get-docker/).
+
 ### Installing Dependencies
 
-Once you have Node.js, PHP, and Composer installed locally, you will need to run `composer install` in the main plugin directory to install the dependencies of the plugin needed to run tests and check coding standards.
+Once you have Node.js, PHP, and Composer installed locally, you will need to install dependencies in the main plugin directory to install the dependencies of the plugin needed to run tests and check coding standards.
```

Comment:
The part "you will need to install dependencies in the main plugin directory to install the dependencies of the plugin needed to run tests" needs revision.